src/sdk/pynni/nni/nas/pytorch/fixed.py

```diff
@@ -41,18 +41,18 @@ def sample_final(self):
         return self._fixed_arc
 
 
-def _encode_tensor(data, device):
+def _encode_tensor(data):
     if isinstance(data, list):
         if all(map(lambda o: isinstance(o, bool), data)):
-            return torch.tensor(data, dtype=torch.bool, device=device)  # pylint: disable=not-callable
+            return torch.tensor(data, dtype=torch.bool)  # pylint: disable=not-callable
```

Comment:
why remove all device?

Reply:
NNI no longer handles device for fixed architecture. Actually in some cases, moving architectures to GPU might cause issues. (see ProxylessNAS)